src/LoadingCollection.ts

```diff
@@ -9,8 +10,9 @@ class LoadingCollection {
    * @param loadArray       Promise that resolves once the array has finished loading
    * @param existingContent optionally set the elements that are already known, for random access
    */
-  static create (loadArray: Promise<Array<ResourceInterface> | undefined>, existingContent: Array<ResourceInterface> = []): Array<ResourceInterface> {
-    // if Promsise resolves to undefined, provide empty array
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
there's no any? can we use [reportUnusedDisableDirectives](https://eslint.org/docs/latest/use/configure/rules#report-unused-eslint-disable-comments) to catch these?